doc/source/tune/examples/index.rst

```diff
@@ -1,24 +1,18 @@
 .. _tune-examples-ref:
+.. _tune-recipes:
 
 =================
 Ray Tune Examples
 =================
 
-.. tip:: Check out :ref:`the Tune User Guides <tune-guides>` To learn more about Tune's features in depth.
-
-.. _tune-recipes:
+.. toctree::
+    :hidden:
```

Comment:
Non-blocking, but how about we remove the `:hidden:` flag here? The page seems a little bare right now

<img width="1197" alt="image" src="https://github.com/ray-project/ray/assets/3967392/50407009-dcf2-4595-bcf4-075140cf9b60">